Jenkinsfile

```diff
@@ -14,66 +14,69 @@ elifePipeline {
             dockerComposeBuild(commit)
         }
 
-        stage 'Project tests', {
-            def actions = [
-                'lint': {
-                    withCommitStatus({
-                        sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_app_lint app npm run lint"
-                    }, 'lint', commit)
-                },
-                'test': {
-                    withCommitStatus({
-                        sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_app_test app npm test"
-                    }, 'test', commit)
-                },
-                'test:dependencies': {
-                    withCommitStatus({
-                        sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_app_test_dependencies app npm run test:dependencies"
-                    }, 'test:dependencies', commit)
-                },
-            ]
+        // stage 'Project tests', {
```

Comment:
all the unit tests are commented out, I guess it's for faster feedback on the later `Browser tests` stage; to be reintroduced

Reply:
Done